### PR TITLE
Missing Data Library items in the Dutch version

### PR DESCRIPTION
**Pull request by FransMeerhoff**
- Switching to the Dutch Data Library some items in the Bain samples are missing.
see https://github.com/jasp-stats/jasp-test-release/issues/508

### CODE REVIEW

File: Resources/Data Sets/index_nl.json [hunk 1]
```diff
@@ -618,6 +618,13 @@
           "description": "BuildIt's monetaire audit populatie bestaande uit 3500 transacties. <br><br> Het voorbeeldbestand in JASP laat zien hoe monetaire audit workflow kan worden toegepast. <br><br> <i>Dataset gebruikt in Derks, de Swart, Wagenmakers, Wille & Wetzels (under review)<\/i>",
           "kind": "file",
           "associated_datafile": "BuildIt_Monetary.csv"
+        },
+        {
+          "name": "Sino Forest",
+          "path": "sinoForest.jasp",
```
Comment (koenderks) [inline Resources/Data Sets/index_nl.json:624]: Het voorbeeldbestand in JASP

File: Resources/Data Sets/index_nl.json [hunk 2]
```diff
@@ -664,20 +671,70 @@
...
+          "name": "ANCOVA",
+        "path": "bainAncova.jasp",
+        "description":"Dit JASP-bestand toont een Bain ANCOVA toets met behulp van de Sesamstraat gegevens. <br> <br> <i> Gegevensset opgenomen in het bain R-pakket. <\ / I>",
+        "kind": "file",
+        "associated_datafile": "sesame.csv"
+      },
+      {
+          "name": "Regressie",
+        "path": "bainRegression.jasp",
+        "description": "Dit JASP-bestand toont een Bain-regressie met behulp van de Sesamstraat gegevens. <br> <br> <i> Gegevensset opgenomen in het bain R-pakket. <\ / I>",
```
Comment (koenderks) [inline Resources/Data Sets/index_nl.json]: Bain regressie

**Review by koenderks — CHANGES_REQUESTED** [review]

File: Resources/Data Sets/index_nl.json (hunk 1 above)
Comment (FransMeerhoff) [reply Resources/Data Sets/index_nl.json:624]: OK

File: Resources/Data Sets/index_nl.json (hunk 2 above)
Comment (FransMeerhoff) [reply Resources/Data Sets/index_nl.json]: OK

**Comment by TimKDJ** [comment]
All good @koenderks?

**Review by koenderks — APPROVED** [review]
Looks good to me

**Pull request merged**